### PR TITLE
Add support for provisioner-less base boxes

### PR DESCRIPTION
**Pull request by schisamo**
Idempotently ensure the desired version of the Omnibus Chef MSI is installed. We install Chef during the `run_command` stage so we can use the sandbox to transfer `install.ps1` (basically a Window's version of `install.sh`).

Tested with a Chef-less box produced from @misheska's Packer templates:
https://github.com/misheska/basebox-packer/tree/master/template

### CODE REVIEW

File: lib/kitchen/provisioner/windows_chef_zero.rb [hunk 1]
```diff
@@ -20,16 +20,31 @@ class WindowsChefZero < ChefZero
...
 
+      # It would be difficult to make the existing
+      # `Kitchen::Provisioner::ChefBase#install_command`
+      # Windows-friendly so we'll just make it no-op.
+      def install_command
+      end
+
       def create_sandbox
         super
+        prepare_install_ps1
```
Comment (portertech) [inline lib/kitchen/provisioner/windows_chef_zero.rb]: shouldn't the conditional `if` just be here?

Comment (schisamo) [inline lib/kitchen/provisioner/windows_chef_zero.rb]: You thinking something like this:

``` ruby
prepare_install_ps1 if config[:require_chef_omnibus]
```

Comment (portertech) [inline lib/kitchen/provisioner/windows_chef_zero.rb]: Yes, not a fan of the `return` no-op pattern.

**Comment by schisamo** [comment]
@portertech OK should be G2G now.

**Comment by schisamo** [comment]
@portertech Anything else that needs to be addressed in this PR?

**Comment by cwjohnston** [comment]
+1 for merging this

**Comment by portertech** [comment]
@cwjohnston I'll assume you will be using this tomorrow ;)

**Comment by portertech** [comment]
:+1:

**Pull request merged**